src/main/kotlin/org/rust/lang/core/types/TyFingerprint.kt

```diff
@@ -42,6 +42,7 @@ data class TyFingerprint constructor(
                 }
                 is RsArrayType -> TyFingerprint("[T]")
                 is RsFnPointerType -> TyFingerprint("fn()")
+                is RsImplTraitType -> TyFingerprint("Anon")
```

Comment:
In the case of `impl Trait` we don't need to change `TyFingerprint`.
`TyFingerprint` is used as a key for indexing (caching) rust impls (`impl X { }`), and it's impossible to write something like `impl impl Trait {}`, so this line is useless :)

Reply:
Done.